lld/test/ELF/aarch64-reloc-abs16-addend.test

```diff
@@ -0,0 +1,38 @@
+# REQUIRES: aarch64
+# RUN: yaml2obj %s -o %t.o
+# RUN: ld.lld %t.o -o %t.out
+# RUN: llvm-objdump -t %t.out | FileCheck %s
+
+# CHECK:      SYMBOL TABLE:
+# CHECK-NEXT: 0000000000210120 g     F bar	0000000000000000 _start
```

Comment:
The llvm-objdump test doesn't appear useful? The `0000000000210120` address is unrelated and would just cause test update trouble if our section layout changes.

I think we need an `llvm-readelf -x .debug_fname` test instead.